New keyword Press Keys, which allows pressing modifiers keys like CTR…

HelioGuilherme66 opened the pull request:
…L, ALT, SHIFT.

Also corrects documentation and code in Press Key (needs \\KEYCODE two '\'), Documentation needs escaped '\' to show properly.

emanlove committed:
Added 'Clear Text' keyword for clearing the text value of a text entry element.

emanlove committed:
Added 'Clear Text' keyword for clearing the text value of a text entry element.

emanlove committed:
Added 'Locator Should Match X Times' keyword similar and more generic version of 'Xpath Should Match X Times' keyword and test.

emanlove committed:
Added 'Locator Should Match X Times' keyword similar and more generic version of 'Xpath Should Match X Times' keyword and test.

emanlove committed:
Added test for new 'Clear Text' keyword.

emanlove committed:
Added test for new 'Clear Text' keyword.

laulaz committed:
Fix url retrieval

See http://www.w3schools.com/jsref/prop_doc_url.asp
This refs #281

laulaz committed:
Fix url retrieval

See http://www.w3schools.com/jsref/prop_doc_url.asp
This refs #281

laulaz committed:
Fix url retrieval

See http://www.w3schools.com/jsref/prop_doc_url.asp
This refs #281

HelioGuilherme66 committed:
Update create_webdriver.txt

Fix test 'Create Webdriver Creates Functioning WebDriver'. Corrected 'chrome' at ${drivers} and added to {dc_names}.

HelioGuilherme66 committed:
Simple HOWTO run the tests.

HelioGuilherme66 committed:
Update forms_and_buttons.txt

Corrected path separator to variable ${/} at 'Navigate To File Upload Form And Create Temp File To Upload'.

HelioGuilherme66 committed:
Extends non-ASCII key codes to Press Key

Proposed by pandabat on https://github.com/rtomac/robotframework-selenium2library/issues/275 ( Press Key does not support non-ASCII, non-typeable characters such as ARROW_RIGHT #275 ).

HelioGuilherme66 committed:
Extends non-ASCII key codes to Press Key

Proposed by pandabat on https://github.com/rtomac/robotframework-selenium2library/issues/275 ( Press Key does not support non-ASCII, non-typeable characters such as ARROW_RIGHT #275 ).

HelioGuilherme66 committed:
Update _element.py

HelioGuilherme66 committed:
Update _element.py

HelioGuilherme66 committed:
Updated tests to pass in firefox and improve in chrome and phantomjs.

HelioGuilherme66 committed:
Updated tests to pass in firefox and improve in chrome and phantomjs.

HelioGuilherme66 committed:
Reverting unwanted changes.

Renaming of tests and resource to *.robot.
Set tag Regression to non-critical in travis.yml.
Utility shell script to rename *.txt to *.robot.

HelioGuilherme66 committed:
Reverting unwanted changes.

lina1 committed:
add method _find_element_by_partial_text

HelioGuilherme66 committed:
Renaming of tests and resource to *.robot.
Set tag Regression to non-critical in travis.yml.
Utility shell script to rename *.txt to *.robot.

HelioGuilherme66 committed:
Fixes Async tests for IE

HelioGuilherme66 committed:
Fixes Async tests for IE

HelioGuilherme66 committed:
Fixes Cookies tests for IE. Changes to PASS IE-Incompatible tests. Sets some tags.

HelioGuilherme66 committed:
Fixes Cookies tests for IE. Changes to PASS IE-Incompatible tests. Sets some tags.

HelioGuilherme66 committed:
Example for IE browser.

HelioGuilherme66 committed:
Example for IE browser.

HelioGuilherme66 committed:
Adds option 'casesense' to keyword 'Textfield Value Should Be'. Updates Choose File test for IE browser and 'casesense' option. Changes resource.robot to Pass IE incompatible tests.

HelioGuilherme66 committed:
Adds option 'casesense' to keyword 'Textfield Value Should Be'. Updates Choose File test for IE browser and 'casesense' option. Changes resource.robot to Pass IE incompatible tests.

HelioGuilherme66 committed:
Fixes 'Content Assertions', 'Frames', 'Javascript' and 'Multiple Browsers' tests for IE

HelioGuilherme66 committed:
Fixes 'Content Assertions', 'Frames', 'Javascript' and 'Multiple Browsers' tests for IE

HelioGuilherme66 committed:
Removed 'Set Debug LogLevel' from Teardowns. Fixes 'Content Assertions' and 'Javascript' tests for IE

HelioGuilherme66 committed:
Removed 'Set Debug LogLevel' from Teardowns. Fixes 'Content Assertions' and 'Javascript' tests for IE

HelioGuilherme66 committed:
Fixes Chrome tests.
Cookies are being created duplicated. Changed test in Delete Cookie.
Javascript popup windows in Chrome are not naming 'main' parent window, or chromedriver does not accept empty name for 'main' window.

HelioGuilherme66 committed:
Fixes Chrome tests.
Cookies are being created duplicated. Changed test in Delete Cookie.
Javascript popup windows in Chrome are not naming 'main' parent window, or chromedriver does not accept empty name for 'main' window.

HelioGuilherme66 committed:
Using robotstatuschecker.py newest version.

HelioGuilherme66 committed:
Fixes LOG tests. Now only two critical tests fail related with custom loglevel in DEBUG. Tests are Page Should Contain With Custom Log Level, and ... Not Contain ...

HelioGuilherme66 committed:
Fixes LOG tests. Now only two critical tests fail related with custom loglevel in DEBUG. Tests are Page Should Contain With Custom Log Level, and ... Not Contain ...

HelioGuilherme66 committed:
Fixes Sizzle tests by using the local jQuery file. Bypasses Chrome, all OK in Firefox and IE. Updates jQuery references from 1.9.1 to 1.11.1.

HelioGuilherme66 committed:
Fixes Sizzle tests by using the local jQuery file. Bypasses Chrome, all OK in Firefox and IE. Updates jQuery references from 1.9.1 to 1.11.1.

HelioGuilherme66 committed:
Fixes Sizzle tests by using the local jQuery file. Bypasses Chrome, all OK in Firefox and IE. Updates jQuery references from 1.9.1 to 1.11.1.

eweitz committed:
QAR-48165: Added support for negative indexes in Robot keywords

eweitz committed:
QAR-48165: Added support for negative indexes in Robot keywords

eweitz committed:
QAR-48165: Added tests for negative indexes, tidied code

eweitz committed:
QAR-48165: Added tests for negative indexes, tidied code

eweitz committed:
QAR-48165: Removed unnecessary import of jQuery

eweitz committed:
QAR-48165: Removed unnecessary import of jQuery

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/laurentbristiel/robotframework-selenium2library
add links to INSTALL and BUILD files (#318)

HelioGuilherme66 committed:
New keyword Press Keys, allows to send keys with Shift, Control or Alt modifiers.

HelioGuilherme66 committed:
New keyword Press Keys, allows to send keys with Shift, Control or Alt modifiers.

HelioGuilherme66 committed:
New keyword Press Keys, allows to send keys with Shift, Control or Alt modifiers.

HelioGuilherme66 committed:
Fixes messages indexes in log tests 'Content Assertions'.

HelioGuilherme66 committed:
Fixes messages indexes in log tests 'Content Assertions'.

HelioGuilherme66 committed:
First working examples.

HelioGuilherme66 committed:
First working examples.

HelioGuilherme66 committed:
First working examples for keyword Press Keys.

ombre42 committed:
doc updates related to loglevels and logging

ombre42 committed:
doc updates related to loglevels and logging

ombre42 committed:
log level trace not supported

ombre42 committed:
log level trace not supported

HelioGuilherme66 committed:
Fix local jquery, unit test updates statuschecker.py to latest version.

HelioGuilherme66 committed:
Fix local jquery, unit test updates statuschecker.py to latest version.

Full list of Keys. Improved tests to pass in Chrome. Backspace, Arrows and Delete test.

Set JQuery to use local library. Recover test_capabilities_attribute_not_modified

HelioGuilherme66 committed:
Fix local jquery, unit test updates statuschecker.py to latest version.

Full list of Keys. Improved tests to pass in Chrome. Backspace, Arrows and Delete test.

HelioGuilherme66 committed:
Fix local jquery, unit test updates statuschecker.py to latest version.

Full list of Keys. Improved tests to pass in Chrome. Backspace, Arrows and Delete test.

Set JQuery to use local library. Recover test_capabilities_attribute_not_modified

HelioGuilherme66 committed:
Fixes LOG tests. All tests PASSED with firefox and chrome. Fourty tests FAILED with Opera (remote Webdriver).

HelioGuilherme66 committed:
Fixes LOG tests. All tests PASSED with firefox and chrome. Fourty tests FAILED with Opera (remote Webdriver).

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

HelioGuilherme66 committed:
Full list of Keys. Improved tests to pass in Chrome. Backspace, Arrows and Delete test.

HelioGuilherme66 committed:
Set JQuery to use local library. Recover test_capabilities_attribute_not_modified

HelioGuilherme66 committed:
Set JQuery to use local library. Recover test_capabilities_attribute_not_modified

emanlove committed:
Moved 'Locator Should Match X Times' acceptance test.

Moved test from test/acceptance/keywords/elements.txt to
test/acceptance/keywords/content_assertions.txt.

emanlove committed:
Moved 'Locator Should Match X Times' acceptance test.

Moved test from test/acceptance/keywords/elements.txt to
test/acceptance/keywords/content_assertions.txt.

HelioGuilherme66 committed:
Ignoring Ghostdriver.log

Ignoring Ghostdriver.log

emanlove committed:
Renamed new 'Clear Text' keyword to 'Clear Element Text'

Renamed keyword to match corresponding and already existing keyword in the
Selenium2Library Java port. Update acceptance tests and removed invalid tests.

emanlove committed:
Renamed new 'Clear Text' keyword to 'Clear Element Text'

Renamed keyword to match corresponding and already existing keyword in the
Selenium2Library Java port. Update acceptance tests and removed invalid tests.

HelioGuilherme66 committed:
Fixes problems with Chrome 37 under Windows 7.

HelioGuilherme66 committed:
Add favicon.icon to html tests folders.

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key
Fix testserver chrome 37 problem in windows.
Adds favicon.ico to html tests.

HelioGuilherme66 committed:
merged from rtomac

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

HelioGuilherme66 committed:
Fixes table_resource.robot file name.

HelioGuilherme66 committed:
Fixes table_resource.robot file name.

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

HelioGuilherme66 committed:
Merge branch 'expand_press_key' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

lina1 committed:
Update test_elementfinder.py

add test_find_by_partial_link_text

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/lina1/robotframework-selenium2library

ombre42 committed:
Document partial link strategy

ombre42 committed:
Document partial link strategy

ombre42 committed:
add default prefix and support prefixes with spaces

ombre42 committed:
add default prefix and support prefixes with spaces

ombre42 committed:
add default prefix and support prefixes with spaces

ombre42 committed:
fix unit test for bad prefixes

ombre42 committed:
fix unit test for bad prefixes

ombre42 committed:
Pass window.id through if not None

ombre42 committed:
Pass window.id through if not None

ombre42 committed:
Pass window.id through if not None

ombre42 committed:
thank the thankless

ombre42 committed:
thank the thankless

ombre42 committed:
improve doc for default locator prefix

ombre42 committed:
improve doc for default locator prefix

ombre42 committed:
fixes #200

Totally unrelated change I know.

ombre42 committed:
typo in default strategy doc

ombre42 committed:
typo in default strategy doc

emanlove committed:
Removed unnecessary check on return value from _element_find.

Independent from Press Key. More tests.

emanlove committed:
Removed unnecessary check on return value from _element_find.

ombre42 committed:
update changelog

ombre42 committed:
add examples of return from JavaScript

ombre42 committed:
add examples of return from JavaScript

HelioGuilherme66 committed:
Fixes tests in Chrome and IE.

HelioGuilherme66 committed:
Fixes tests in Chrome and IE.

HelioGuilherme66 committed:
Merged with origin/rtomac

HelioGuilherme66 committed:
Merge remote-tracking branch 'origin/expand_press_key' into expand_press_key

HelioGuilherme66 committed:
Independent from Press Key. More tests.

HelioGuilherme66 committed:
Merge branch 'expand_press_key' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

Conflicts:
	CHANGES.rst

HelioGuilherme66 committed:
Clear Element Text fix. No speed slowdown.

HelioGuilherme66 committed:
Clear Element Text fix. No speed slowdown.

Removed test step in Press Keys for Firefox.

remove tag inprogress, capital letters in test

HelioGuilherme66 committed:
remove tag inprogress, capital letters in test

HelioGuilherme66 committed:
Removed test step in Press Keys for Firefox.

HelioGuilherme66 committed:
Removed test step in Press Keys for Firefox.

HelioGuilherme66 committed:
Merge branch 'expand_press_key' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

Conflicts:
	test/acceptance/keywords/keyboard.robot

HelioGuilherme66 committed:
Fixed AsyncJS messages, renamed README.rst, added reverse option to util.sh.

HelioGuilherme66 committed:
Fixed AsyncJS messages, renamed README.rst, added reverse option to util.sh.

HelioGuilherme66 committed:
Fixed AsyncJS messages, renamed README.rst, added reverse option to util.sh.

HelioGuilherme66 committed:
Removed Ctrl-A from test, breaking with firefox.

HelioGuilherme66 committed:
Removed Ctrl-A from test, breaking with firefox.

HelioGuilherme66 committed:
Corrected AsyncJS messages.

HelioGuilherme66 committed:
Corrected AsyncJS messages.

HelioGuilherme66 committed:
Corrected AsyncJS messages.

HelioGuilherme66 committed:
Fixes webdriver messages. Reverse option in util.sh.

HelioGuilherme66 committed:
Fixes webdriver messages. Reverse option in util.sh.

HelioGuilherme66 committed:
Fixes webdriver messages. Reverse option in util.sh.

HelioGuilherme66 committed:
Merge branch 'expand_press_key' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

Conflicts:
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/keyboard.robot

HelioGuilherme66 committed:
Commented Control-Z test.

HelioGuilherme66 committed:
Commented Control-Z test.

HelioGuilherme66 committed:
Fixes webdriver messages.

HelioGuilherme66 committed:
Merge remote-tracking branch 'origin/master' into expand_press_key

Conflicts:
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/lists.robot

HelioGuilherme66 committed:
Fixed log level messages indexes.

HelioGuilherme66 committed:
Fixed log level messages indexes.

HelioGuilherme66 committed:
Fixes log level messages indexes

HelioGuilherme66 committed:
Fixes webdriver messages for Firefox

Changed the name of the test suite to run in last order, because Firefox would change to Open separate pages instead of Tabbed.

HelioGuilherme66 committed:
Fixes webdriver messages for Firefox

HelioGuilherme66 committed:
Fixes webdriver messages for Firefox

HelioGuilherme66 committed:
Merge branch 'master' into expand_press_key

HelioGuilherme66 committed:
Changed the name of the test suite to run in last order, because Firefox would change to Open separate pages instead of Tabbed.

HelioGuilherme66 committed:
Merge branch 'expand_press_key' of https://github.com/HelioGuilherme66/robotframework-selenium2library into expand_press_key

HelioGuilherme66 committed:
Merge from rtomac

HelioGuilherme66 committed:
Update .robot files from last merge.

HelioGuilherme66 committed:
Update .robot files from last merge.

HelioGuilherme66 committed:
Merge remote-tracking branch 'origin/master' into expand_press_key

Conflicts:
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/content_assertions.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Expands Press Key by adding a new keyword Press Keys which allows to press simultaneous keys, like Alt, Shift, Control and alphabetic keys. It also allows to send sequence of keys like they are being pressed.
Relevant changes to files are at _element.py and new test file zkeyboard.robot which uses a new HTML file long_page.html.

HelioGuilherme66 committed:
Expands Press Key by adding a new keyword Press Keys which allows to press simultaneous keys, like Alt, Shift, Control and alphabetic keys. It also allows to send sequence of keys like they are being pressed.
Relevant changes to files are at _element.py and new test file zkeyboard.robot which uses a new HTML file long_page.html.

HelioGuilherme66 committed:
New keyword Press Keys, which allows pressing modifiers keys like CTRL, ALT, SHIFT.

HelioGuilherme66 committed:
New keyword Press Keys, which allows pressing modifiers keys like CTRL, ALT, SHIFT.

just-be-dev commented on src/Selenium2Library/keywords/_element.py:
```diff
@@ -480,13 +480,16 @@ def press_key(self, locator, key):
-        | Press Key | login_button | \\\\13 | # ASCII code for enter key |
+        | Press Key | text_field   | q                |                                               |
+        | Press Key | login_button | \\\\13           | # ASCII code for enter key                    |
+        | Press Key | nav_console  | \\\\\\\\ARROW_UP | # selenium.webdriver.common.keys ARROW_UP KEY |
```
Why are there so many escapes here? Is it necessary?

HelioGuilherme66 replied:
The escapes were added to be correctly seen in HTML pages and RIDE context help.
They are double backslash for ASCII codes, and quadruple for KEY NAME, and then escaped for being shown.

just-be-dev replied:
Ohhh, well that makes sense then. Sorry about that, ha.

just-be-dev commented on src/Selenium2Library/keywords/_element.py:
```diff
@@ -734,6 +791,79 @@ def _map_ascii_key_code_to_key(self, key_code):
+            'TAB': Keys.TAB,
+            'UP': Keys.UP
+        }
+        key = map.get(key_name)
```
I know I mentioned this in the last PR, but you should probably replace this whole map with 

``` python
try:
  return getattr(Keys, key_name)
except:
  # emit proper error here
```

That way you don't have to define this whole redundant map. It's smaller and less maintenance as well. 

Oh, and you should probably give a descriptive error on the assert just so people know why it failed.

HelioGuilherme66 replied:
I used your suggestion. It works well.
I am having trouble with the commit.

just-be-dev replied:
What kind of trouble? I'll assist as best I can.

HelioGuilherme66 committed:
Removed map for Press Keys and corrected ValueErrors output.

HelioGuilherme66 committed:
Removed map for Press Keys and corrected ValueErrors output.

Fix doc in Press Keys

First working examples for keyword Press Keys.

HelioGuilherme66 committed:
Removed map for Press Keys and corrected ValueErrors output.

Fix doc in Press Keys

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

Conflicts:
	src/Selenium2Library/keywords/_element.py
	test/acceptance/keywords/elements.robot
	test/acceptance/keywords/run_on_failure.txt

just-be-dev commented:
Sorry it's taking me so long to review this. I haven't forgotten you, I'll get to it soon.

HelioGuilherme66 committed:
Fix doc in Press Keys

HelioGuilherme66 committed:
Fix doc in Press Keys

HelioGuilherme66 committed:
Trying to fix merging mess :(

HelioGuilherme66 committed:
A copy of resource.robot with old extension txt.

HelioGuilherme66 committed:
Merge branch 'keyword_press_keys' of https://github.com/HelioGuilherme66/robotframework-selenium2library into keyword_press_keys

Conflicts:
	src/Selenium2Library/keywords/_element.py

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

Conflicts:
	test/acceptance/windows.robot
	test/resources/html/jquery.html

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

HelioGuilherme66 committed:
Merge branch 'master' into keyword_press_keys

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library into keyword_press_keys

Conflicts:
	test/acceptance/create_webdriver.robot
	test/acceptance/keywords/__init__.robot
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/checkbox_and_radio_buttons.robot
	test/acceptance/keywords/content_assertions.robot
	test/acceptance/keywords/forms_and_buttons.robot
	test/acceptance/keywords/frames.robot
	test/acceptance/keywords/javascript.robot
	test/acceptance/keywords/lists.robot
	test/acceptance/keywords/mouse.robot
	test/acceptance/keywords/navigation.robot
	test/acceptance/keywords/screenshots.robot
	test/acceptance/keywords/textfields.robot
	test/acceptance/locators/__init__.robot
	test/acceptance/locators/sizzle.robot
	test/acceptance/multiple_browsers.robot
	test/acceptance/open_and_close.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

Conflicts:
	test/acceptance/create_webdriver.robot
	test/acceptance/keywords/__init__.robot
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/checkbox_and_radio_buttons.robot
	test/acceptance/keywords/content_assertions.robot
	test/acceptance/keywords/forms_and_buttons.robot
	test/acceptance/keywords/frames.robot
	test/acceptance/keywords/javascript.robot
	test/acceptance/keywords/lists.robot
	test/acceptance/keywords/mouse.robot
	test/acceptance/keywords/navigation.robot
	test/acceptance/keywords/screenshots.robot
	test/acceptance/keywords/textfields.robot
	test/acceptance/locators/__init__.robot
	test/acceptance/locators/sizzle.robot
	test/acceptance/multiple_browsers.robot
	test/acceptance/open_and_close.robot
	test/acceptance/resource.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library

Conflicts:
	test/acceptance/create_webdriver.robot
	test/acceptance/keywords/__init__.robot
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/checkbox_and_radio_buttons.robot
	test/acceptance/keywords/content_assertions.robot
	test/acceptance/keywords/forms_and_buttons.robot
	test/acceptance/keywords/frames.robot
	test/acceptance/keywords/javascript.robot
	test/acceptance/keywords/lists.robot
	test/acceptance/keywords/mouse.robot
	test/acceptance/keywords/navigation.robot
	test/acceptance/keywords/screenshots.robot
	test/acceptance/keywords/textfields.robot
	test/acceptance/locators/__init__.robot
	test/acceptance/locators/sizzle.robot
	test/acceptance/multiple_browsers.robot
	test/acceptance/open_and_close.robot
	test/acceptance/resource.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Fix merge

HelioGuilherme66 committed:
Merge branch 'expand_press_key' into keyword_press_keys

Conflicts:
	src/Selenium2Library/keywords/_element.py
	test/acceptance/create_webdriver.robot
	test/acceptance/keywords/navigation.robot
	test/acceptance/keywords/textfields.robot

HelioGuilherme66 committed:
Fix merge

HelioGuilherme66 committed:
fix merge

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/HelioGuilherme66/robotframework-selenium2library

Conflicts:
	test/acceptance/keywords/content_assertions.robot
	test/acceptance/keywords/forms_and_buttons.robot
	test/acceptance/keywords/lists.robot
	test/acceptance/keywords/screenshots.robot
	test/acceptance/locators/__init__.robot
	test/acceptance/open_and_close.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Merge branch 'master' into keyword_press_keys

Conflicts:
	test/acceptance/create_webdriver.robot
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/open_and_close.robot
	test/acceptance/resource.robot
	test/acceptance/windows.robot

HelioGuilherme66 committed:
Fixes async messages for Chrome, Phantomjs and IE.

dpsfrishberg committed:
Removing unused import

I was looking at this file and noticed an unused import.

Removing unused import

Another unused import here.

HelioGuilherme66 committed:
Fixes tests in Chrome and IE.

Conflicts:
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/textfields.robot

HelioGuilherme66 committed:
Fixes tests in Chrome and IE.

Conflicts:
	test/acceptance/keywords/async_javascript.robot
	test/acceptance/keywords/textfields.robot

HelioGuilherme66 committed:
Fixed AsyncJS messages, renamed README.rst, added reverse option to util.sh.

Fixes webdriver messages. Reverse option in util.sh.

Corrected AsyncJS messages.

HelioGuilherme66 committed:
Corrects resource.robot import in elements.robot test.
Removes tests *.txt after renaming to *.robot.

HelioGuilherme66 committed:
Merge branch 'master' of https://github.com/rtomac/robotframework-selenium2library into keyword_press_keys

HelioGuilherme66 committed:
Big merge fix.
Activate JQuery tests for Chrome.

Commit:
Merge branch 'master' into keyword_press_keys

HelioGuilherme66 commented:
A new PR will be done.
